From 56a84fbabed530150787cad4f98aad414a0fee57 Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Thu, 16 Oct 2014 22:18:03 +0800
Subject: [PATCH] MLK-9618-10 usb: chipidea: otg: change TA_WAIT_VFAL to be
 500ms

500ms is enough for vbus fall to VOTG_VBUS_LKG(0.7v, OTG device or EH
Leakage Voltage).

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit 41295ac496aa7697585e65223797e0cb4e5fb153)
---
 drivers/usb/chipidea/otg_fsm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/chipidea/otg_fsm.h b/drivers/usb/chipidea/otg_fsm.h
index 2b4d83b..3eec4d2 100644
--- a/drivers/usb/chipidea/otg_fsm.h
+++ b/drivers/usb/chipidea/otg_fsm.h
@@ -25,7 +25,7 @@
 					 * ->DC Electrical Timing
 					 */
 /* Wait for VBUS Fall  */
-#define TA_WAIT_VFALL        (1000)	/* a_wait_vfall: section 7.1.7
+#define TA_WAIT_VFALL        (500)	/* a_wait_vfall: section 7.1.7
 					 * a_wait_vfall_tmr: section: 7.4.5.2
 					 */
 /* Wait for B-Connect */
-- 
1.8.0

